sermant-agentcore/sermant-agentcore-implement/src/main/java/io/sermant/implement/service/xds/handler/XdsHandler.java

```diff
@@ -143,9 +159,17 @@ public void onError(Throwable throwable) {
                 if (countDownLatch != null) {
                     countDownLatch.countDown();
                 }
-                client.updateChannel();
-                subscribe(requestKey, null);
-                LOGGER.log(Level.SEVERE, "An error occurred in Xds communication with istiod.", throwable);
+                initExecutor();
+                executor.submit(() -> {
+                    try {
+                        Thread.sleep(DELAY_TIME);
+                    } catch (InterruptedException e) {
+                        LOGGER.log(Level.WARNING, "An error occurred in thread sleeping.", e);
+                    }
+                    client.updateChannel();
+                    subscribe(requestKey, null);
+                    LOGGER.log(Level.SEVERE, "An error occurred in Xds communication with istiod.", throwable);
+                });
```

Comment:
this log should be out of executor. submit

Reply:
done